modules/kn-service-offline-about.adoc

```diff
@@ -0,0 +1,21 @@
+// Module included in the following assemblies:
+//
+// serverless/cli_reference/kn-offline-services.adoc
+
+[id="kn-service-offline-about_{context}"]
+= About offline mode
+
+Normally, when you execute `kn service` commands, the changes immediately propagate to the cluster. However, as an alternative, you can execute `kn service` commands in offline mode:
+
+. When you create a service in offline mode, no changes happen on the cluster. Instead, the only thing that happens is the creation of the service descriptor file on your local machine.
+. After the descriptor file is created, you can manually modify it and track it in a version control system.
+// Once `update` works, add it here and make it into a list
+. Finally, you can propagate changes to the cluster. For that, you use commands such as `kn service create -f`, `kn service apply -f`, or `oc apply -f` on the descriptor files.
```

Comment:
```suggestion
. You can propagate changes to the cluster by using the `kn service create -f`, `kn service apply -f`, or `oc apply -f` commands on the descriptor files.
```

Reply:
Suggestion for readability

Reply:
Used the suggestion, but kept "Finally, " - it helps draw the line between the previous steps (which were offline mode) and this final step which is not offline mode anymore.